docker/pusher.go

```diff
@@ -67,7 +67,7 @@ func (n *Pusher) Push(f bosonFunc.Function) (digest string, err error) {
 	return
 }
 
-var digestRE = regexp.MustCompile(`digest:\s+sha256:(?P<Digest>\w{64})`)
+var digestRE = regexp.MustCompile(`digest:\s+(sha256:\w{64})`)
```

Comment:
Just want to confirm that this is the change discussed with @matejvasek last week.

Reply:
It should be OK. At worst it will cause merge conflict.

Reply:
Yes, this is the one.  As Matej mentions, I am expecting a merge conflict to resolve or perhaps a rebase.